scripts/scil_tractogram_convert_hdf5_to_trk.py

```diff
@@ -59,8 +59,9 @@ def _build_arg_parser():
                             'interest.\nEquivalent to adding any --edge_keys '
                             'node_LABEL2 or LABEL2_node.')
 
-    p.add_argument('--save_empty', metavar='labels_list', dest='labels_list',
-                   help='Save empty connections. Then, the list of possible '
+    p.add_argument('--save_empty', nargs='?', metavar='labels_list',
+                   dest='labels_list', const=True,
+                   help='Save empty connections. The list of possible '
```

Comment:
I would add in the description that the labels_list is now facultative.

Reply:
Why not change the dest to `labels_file` instead. It sounds like `args.labels_list` should be a list.